.circleci/config.yml

```diff
@@ -295,3 +322,7 @@ workflows:
             branches:
               only:
                 - master
+  rebase-and-unit-test: #note: this workflow is triggered remotely on merchant branches when a PR is submitted
```

Comment:
skip this on master?

Reply:
added condition to test-merchant-branches script

Reply:
i mean filter out master here in the circle config